### PR TITLE
A001 check

### PR DESCRIPTION
**Pull request by dmius**
First version of system info check

### CODE REVIEW

**Comment by NikolayS** [comment]
looks good,
let's add:

1) more pieces -- see my comments in https://github.com/postgres-ai/postgres-health-check/issues/6
2) "raw" output from `lscpu` in JSON. It's worth to have both raw output from commands, and decomposed values.

File: resources/checks/A001_system_info.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#Collect system information
+HOSTS=(localhost)
```
Comment (Nastradamus) [inline resources/checks/A001_system_info.sh]: We should use $HOST global variable, not localhost.

File: resources/checks/A001_system_info.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+function get_mem_info() {
+  local host=$1
+  local res=""
+  #local mem_info="$(ssh "$host" "cat /proc/meminfo" | sed 's/"/\\"/g')"
+  local mem_info="$(cat /proc/meminfo | sed 's/"/\\"/g')"
+
+  local res_obj="{\"cmd2check\": \"cat /proc/meminfo\""
+  while read -r line; do
+    arg=$(echo "$line" | sed 's/:.*$//g' )
```
Comment (Nastradamus) [inline resources/checks/A001_system_info.sh]: we should remove sed

File: resources/checks/A001_system_info.sh [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+CPU_INFO=""
+MEM_INFO=""
+OS_INFO=""
+DISK_INFO=""
+CTL_INFO=""
+
+function get_cpu_info() {
+  local host=$1
+  local res=""
+  #local cpu_info="$(ssh "$host" "lscpu" | sed 's/"/\\"/g')"
```
Comment (Nastradamus) [inline resources/checks/A001_system_info.sh]: Where is a ssh connection? It is completly don't work with remote server.

**Comment by Nastradamus** [comment]
Failed with master branch:
```
[vyagofarov@mac:~/git/healthcheck] $ ./check -h localhost -p 5432 --username vyagofarov --dbname postgres --project dummy
[2018-11-21T11:37:04+0300] ========== Perform check: ==========
[2018-11-21T11:37:04+0300] ./resources/checks/A001_system_info.sh
[2018-11-21T11:37:04+0300]
./resources/checks/A001_system_info.sh: line 13: lscpu: command not found
cat: /proc/meminfo: No such file or directory
df: option requires an argument -- T
usage: df [-b | -H | -h | -k | -m | -g | -P] [-ailn] [-T type] [-t] [filesystem ...]
./resources/checks/A001_system_info.sh: line 53: hostnamectl: command not found
[2018-11-21T11:37:04+0300] ========== End of check ===========
[2018-11-21T11:37:04+0300] check 'A001_system' is finished
[2018-11-21T11:37:04+0300] Artifacts saved at: './artifacts/dummy/2018_11_21T11_37_04_+0300/A001_system.json'
```

**Comment by Nastradamus** [comment]
It works great!

**Pull request merged**